### DISCUSSION TITLE
Puppeteering Tauri window

### DISCUSSION THREAD

**Discussion by mrceperka** [tauri-apps/tauri#5481] [Q&A]
Hello,

I have **tauri** app with **nodejs sidecar.** I use this sidecar for controlling **google-chrome via puppeteer*.

But this setup might not be ideal.
Considering that tauri window is webview-kind-of-thing it might be possible to open new window on given URL and just control that whole window.

Is it possible?
_I've seen some `window.eval`, but I am not sure if it is possible to control that window for the whole time._

Thanks!

**Comment by Osub** [comment]
+1

**Reply by Osub** [reply]
https://crates.io/crates/puppeteer

**Reply by jarekt** [reply]
what does this have to do with the actual puppeteer?

**Comment by kitABAB** [comment]
yeah i face the problem as you say。i write a puppeteer program and use it as a sidebar 。when the sidebar start ，it open new window and i have no method to prevent it
